src/main/java/co/inventrosoft/springboottask/model/Team.java

```diff
@@ -0,0 +1,26 @@
+package co.inventrosoft.springboottask.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Team {
+    int id;
+    String name;
+    String capitan;
```

Comment:
There is no such word in English `capitan`.
There is `captain`.
https://www.oxfordlearnersdictionaries.com/definition/english/captain_1?q=captain